addons/point_of_sale/models/pos_order.py

```diff
@@ -565,6 +565,26 @@ def create_from_ui(self, orders, draft=False):
 
         return self.env['pos.order'].search_read(domain = [('id', 'in', order_ids)], fields = ['id', 'pos_reference'])
 
+    @api.model
+    def check_synced_orders(self, orders):
+        """
+        Check the list of unpaid orders in the point of sale and return the completed orders.
+
+        Look through the list of unpaid orders in the current point of sale and find the
+        ones that are completed. Whenever the ticket screen is accessed, the returned orders
+        need to be removed.
```

Comment:
It's really nice of you to write some documentations :+1: 
Ideally you should only write about what the function does and how. You should not write a precise description of the context on when and why it is called. 
So basically what this function does is simply filtering out a list of pos orders references and keeping the ones that are not in draft anymore. Nothing else.

The reason why `Whenever the ticket screen is accessed, the returned orders need to be removed.` is bad is because your function can now be called by someone else elsewhere. Consider this as a public tool that you just created